misc/config_tools/schema/checks/rdt_support.xsd

```diff
@@ -53,4 +53,13 @@ This error cannot be fixed by adjusting the configuration. Report a `GitHub issu
     </xs:annotation>
   </xs:assert>
 
+  <xs:assert test="every $vm_name in //vm/name satisfies
+                  if (//RDT_ENABLED = 'y')
+                  then count(//POLICY/VM[text() = $vm_name]) > 0
```

Comment:
Shall we check that all the vCPUs of that VM has a cache allocation policy defined if vCAT is not enabled for this VM?

Reply:
Yes, we could.
We have talk these case and decide to simple it.
Now the CLOS MASK missing  of XMLs which generate by UI is some VM's all VCPUs have no CACHE config.